apps/prairielearn/assets/scripts/instructorQuestionSettingsClient.ts

```diff
@@ -11,6 +19,46 @@ onDocumentReady(() => {
   );
   const saveButton = document.querySelector<HTMLButtonElement>('#save-button');
 
+  new TomSelect('#topic', {
+    valueField: 'name',
+    searchField: ['name'],
+    closeAfterSelect: true,
+    items: [(document.querySelector('#topic') as HTMLSelectElement).dataset.selectedTopic],
```

Comment:
Rather than this custom thing with `data-selected-topic`, you should be able to just specify the `selected` attribute on the `<option>` that's initially selected.

Reply:
Updated to specify `selected` in [98f2dd5](https://github.com/PrairieLearn/PrairieLearn/pull/10868/commits/98f2dd558c3f286228530e2a7899b8db990394be).